engine/org.eclipse.birt.report.engine.emitter.odt/.classpath

```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
-		<attributes>
-			<attribute name="module" value="true"/>
-			<attribute name="limit-modules" value="java.base"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="target/classes"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
+		<attributes>
+			<attribute name="module" value="true"/>
+			<attribute name="limit-modules" value="java.logging,jdk.xml.dom"/>
```

Comment:
should we really use "limit-modules"?

Reply:
In this case: probably no. 
There were already a "limit-modules" in the .classpath file. It was not in affect since Java1.8 was used. When Java-11 was enabled I added bundles to resolve compilation issues. I should have removed limit-modules all together.

Other projects have limit-modules that looks strange and was also probably added earlier. Some hade openj9 modules in them. Strange.

I think that after @gab1one comments about problems with resolution chains of org.w3c.dom.events, we may need to add limit-modules to some bundles, but that is another thing I guess.